tests/ci.sh

```diff
@@ -3,7 +3,15 @@
 # Copyright 2020 Contributors to the Parsec project.
 # SPDX-License-Identifier: Apache-2.0
 
-set -euf -o pipefail
+set -xeuf -o pipefail
```

Comment:
Added `x` so that all executed commands are printed. Otherwise it's a bit annoying to figure out where an error ocurred.